{"name": "test_login[argvals0]", "status": "broken", "statusDetails": {"message": "UnboundLocalError: local variable 'base' referenced before assignment", "trace": "self = <allure_commons._allure.fixture object at 0x000001DA94CB8DC8>, args = ()\nkwargs = {}\n\n    def __call__(self, *args, **kwargs):\n        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)\n    \n        with self:\n>           return self._fixture_function(*args, **kwargs)\n\n..\\..\\..\\..\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\allure_commons\\_allure.py:200: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    @pytest.yield_fixture(scope ='function', autouse=True)\n    def base_fixture():\n        try:\n            prodpath = open(constants.ENVIRONMENT_PROPERTIES)\n            prod.load(prodpath)\n            #prod.list()\n            base = BasePage()\n            objlist.append(base)\n        except Exception:\n            pass\n    \n        yield locals()\n>       base.quit()\nE       UnboundLocalError: local variable 'base' referenced before assignment\n\nconftest.py:26: UnboundLocalError"}, "parameters": [{"name": "argvals", "value": "{'RunMode': 'y', 'Browsername': 'Chrome', 'Username': 'u1', 'Password': 'p1'}"}], "start": 1630604805576, "stop": 1630604805581, "uuid": "d4545c84-500c-4b50-88d9-e9cc191e2c0d", "historyId": "e1032ad29a99f88dcbdb7506d5c2b1b1", "testCaseId": "93a39feb8eb701231e6c069b76eeac98", "fullName": "testcases.test_login.TestLogin#test_login", "labels": [{"name": "tag", "value": "usefixture('base_fixture')"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "WTIN05200385L"}, {"name": "thread", "value": "68792-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_login"}]}